dotnet/src/VectorDataIntegrationTests/AzureAISearchIntegrationTests/Filter/AzureAISearchFilterFixture.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using AzureAISearchIntegrationTests.Support;
+using VectorDataSpecificationTests.Filter;
+using VectorDataSpecificationTests.Support;
+
+namespace AzureAISearchIntegrationTests.Filter;
+
+public class AzureAISearchFilterFixture : FilterFixtureBase<string>
+{
+    protected override TestStore TestStore => AzureAISearchTestStore.Instance;
+
+    // Azure AI search only supports lowercase letters, digits or dashes.
+    protected override string StoreName => "filter-tests";
+
+    public override async Task DisposeAsync()
+        => await base.DisposeAsync();
```

Comment:
What is  the point of overriding this method? Just to be able to hit the breakpoint here?

Reply:
Nope, just a leftover - will remove.